app/controllers/profile_controller.rb

```diff
@@ -9,13 +9,13 @@ def index
     user = current_user if user.nil?
 
     # Determine how the profile should be displayed based on the user_id
-    if user == current_user
-      @stories = Kaminari.paginate_array(get_stories(current_user, false))
+    @stories = if user == current_user
+                 Kaminari.paginate_array(get_stories(current_user, false))
```

Comment:
The two statements inside the conditional are similar. 
Maybe we could have

```ruby
if user == current_user || current_user.allies_by_status(:accepted).include?(user)
  @stories = Kaminari.paginate_array(get_stories(user, false)).page(params[:page])
end
```
better enough, we could put `user == current_user || current_user.allies_by_status(:accepted).include?(user)` in a variable

Reply:
From my suggestion my aim was to do away with the elseif block. That is why I did
```ruby
if user == current_user || current_user.allies_by_status(:accepted).include?(user)
  @stories = Kaminari.paginate_array(get_stories(user, false)).page(params[:page])
end
```

We also don't need a separate method for the pagination